dev-tools/reconfigurator-cli/tests/output/cmds-mupdate-update-flow-stdout

```diff
@@ -404,3 +429,1644 @@ COCKROACH STATUS
+    name: _oximeter-reader._tcp                              (records: 1)
+        SRV  port  9000 353b3b65-20f7-48c3-88f7-495bd5d31545.host.control-plane.oxide.internal
+    name: _repo-depot._tcp                                   (records: 3)
+        SRV  port 12348 2b8f0cb3-0295-4b3c-bc58-4fe88b57112c.sled.control-plane.oxide.internal
+        SRV  port 12348 98e6b7c2-2efa-41ca-b20a-0a4d61102fe6.sled.control-plane.oxide.internal
+        SRV  port 12348 d81c6a84-79b8-4958-ae41-ea46c9b19763.sled.control-plane.oxide.internal
+    name: ad6a3a03-8d0f-4504-99a4-cbf73d69b973.host          (records: 1)
+        AAAA fd00:1122:3344:102::25
+    name: ba4994a8-23f9-4b1a-a84f-a08d74591389.host          (records: 1)
+        AAAA fd00:1122:3344:101::24
+    name: bd354eef-d8a6-4165-9124-283fb5e46d77.host          (records: 1)
+        AAAA fd00:1122:3344:102::26
+    name: d81c6a84-79b8-4958-ae41-ea46c9b19763.sled          (records: 1)
+        AAAA fd00:1122:3344:103::1
+    name: ea5b4030-b52f-44b2-8d70-45f15f987d01.host          (records: 1)
+        AAAA fd00:1122:3344:3::1
+    name: f10a4fb9-759f-4a65-b25e-5794ad2d07d8.host          (records: 1)
+        AAAA fd00:1122:3344:103::21
+    name: f55647d4-5500-4ad3-893a-df45bd50d622.host          (records: 1)
+        AAAA fd00:1122:3344:103::25
+    name: f6ec9c67-946a-4da3-98d5-581f72ce8bf0.host          (records: 1)
+        AAAA fd00:1122:3344:103::23
+    name: ns1                                                (records: 1)
+        AAAA fd00:1122:3344:1::1
+    name: ns2                                                (records: 1)
+        AAAA fd00:1122:3344:2::1
+    name: ns3                                                (records: 1)
+        AAAA fd00:1122:3344:3::1
+
+external DNS:
+  DNS zone: "oxide.example" (unchanged)
+    name: @                                                  (records: 3)
+        NS   ns1.oxide.example
+        NS   ns2.oxide.example
+        NS   ns3.oxide.example
+    name: example-silo.sys                                   (records: 3)
+        A    192.0.2.2
+        A    192.0.2.3
+        A    192.0.2.4
+    name: ns1                                                (records: 1)
+        A    198.51.100.1
+    name: ns2                                                (records: 1)
+        A    198.51.100.2
+    name: ns3                                                (records: 1)
+        A    198.51.100.3
+
+
+
+
+> # Clear the mupdate override on sled 2, signifying that the config has been
+> # applied.
+> sled-set serial2 mupdate-override unset
+set sled d81c6a84-79b8-4958-ae41-ea46c9b19763 mupdate override: 1c0ce176-6dc8-4a90-adea-d4a8000751da -> unset
+
+
+> # Run the planner again. This will cause sled 2's blueprint
+> # remove_mupdate_override to be unset. But no further planning steps will
+> # happen because the target release generation is not new enough.
+> #
+> # TODO: we want to block remove_mupdate_override unsets until the
```

Comment:
Same note as above; wondering if it's fine to keep this as-is and only change the planner's "can we proceed" check.